ts/features/itwallet/identification/screens/ItwIdentificationCieWarningScreen.tsx

```diff
@@ -0,0 +1,100 @@
+import { Linking } from "react-native";
+import { ItwEidIssuanceMachineContext } from "../../machine/provider";
+import { IOStackNavigationRouteProps } from "../../../../navigation/params/AppParamsList";
+import { ItwParamsList } from "../../navigation/ItwParamsList";
+import {
+  OperationResultScreenContent,
+  OperationResultScreenContentProps
+} from "../../../../components/screens/OperationResultScreenContent.tsx";
+import I18n from "../../../../i18n.ts";
+import { useHeaderSecondLevel } from "../../../../hooks/useHeaderSecondLevel.tsx";
+
+export type CieWarningType = "noPin" | "noCie";
+
+export type ItwIdentificationCieWarningScreenNavigationParams = {
+  warning: CieWarningType;
+};
+
+type ScreenProps = IOStackNavigationRouteProps<
+  ItwParamsList,
+  "ITW_IDENTIFICATION_CIE_WARNING"
+>;
+
+export const ItwIdentificationCieWarningScreen = (params: ScreenProps) => {
+  const machineRef = ItwEidIssuanceMachineContext.useActorRef();
+  const { warning } = params.route.params;
+
+  const closeIdentification = () => {
+    machineRef.send({ type: "close" });
```

Comment:
I think we should just pop the screen. No need to close the entire flow

Reply:
@mastro993 If you mean the close button on those screens, I confirmed with the design team that that button should close the flow. The back button behavior issue found by @RiccardoMolinari95 needs to be fixed